pxblue/colors
# Add github blui-ci

Fixes BLUI-5600.



#### Changes proposed in this Pull Request:

- add blui ci github workflow
- remove circleCI config and stopped build in circleCI dashboard
- updated the buildTest.sh to check colorType files



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- optional commit to see jobs run and remove commit
- https://github.com/etn-ccis/blui-colors/actions/runs/8707134562



#### Any specific feedback you are looking for?

-
